src/templates/templates.service.spec.ts

```
@@ -0,0 +1,137 @@
+  });
+
+  describe('remove', () => {
+    it('should mark a clause as inactive', async () => {
+      const clause = { id: '1', isActive: true } as StandardClause;
+
+      repository.findOne.mockResolvedValue(clause);
+      repository.save.mockResolvedValue({ ...clause, isActive: false });
+
+      await service.remove('1');
+
+      expect(repository.save).toHaveBeenCalledWith({ ...clause, isActive: false });
```

ESLint

[error] Replace `·...clause,·isActive:·false` with `⏎········...clause,⏎········isActive:·false,⏎·····`

(prettier/prettier)